examples/python/IRD_stream.py

```diff
@@ -76,36 +76,39 @@
 # flatten
 target.flatten()
 # assign reference spectra & resample
-target.dispcor()
+target.dispcor(master_path=thar.anadir)
 
 ### FLAT (for blaze function) ###
 flat.trace = trace_mmf
 if flat.band == 'h':
     flat.clean_pattern(extin='', extout='_cp', hotpix_mask=hotpix_mask)
 flat.imcomb = True # median combine
 flat.flatten()
-flat.dispcor()
+flat.dispcor(master_path=thar.anadir)
 
 # combine & normalize
-target.normalize1D(flatid=flat.streamid)
+target.normalize1D(flatid=flat.streamid,master_path=flat.anadir)
 
-"""### mmfmmf (test) ###
+"""
+### FOR RV MEASUREMENTS ###
```

Comment:
The commented-out part below is the part that creates some files needed for RV measurements. So, you can ignore the change.